database/schema.sql

```diff
@@ -597,45 +597,30 @@ CREATE TABLE IF NOT EXISTS dbo.sccv_request_audit(
 
```

Comment:
I'm not certain but when we update the schema I think we  don't just delete columns from existing tables.

Everything has to be a migration so we would add new lines to alter existing by deleting columns?

The current system doing it manually is awful anyway but I'm pretty sure we don't run the whole script just the changes we make. Maybe verify this with @LBHTKarki

Reply:
Hi John, I was under the impression that the db schema updates had their own place in the `database/manual-updates` folder and that the `schema.sql` file is used only for the test database .

I also wonder at this point what's the purpose of `seeds.sql`, what's is lifecycle?

@LBHTKarki do you have any extra info about this?

Reply:
@mapleleafjack the current process for updating the schema is pretty awful like @JohnFarrellDev said. However it is important to keep writing the schema file using the migration approach i.e. altering tables, columns etc. rather than replacing the table definitions in place.

Writing the changes this way ensures we can always apply the changes to any environment safely without having to touch data (in most cases, there are exceptions of course). Say you add a new column to a table. If the table definition is changed in place and you were to write a manual database update for that in the `database/manual-updates` folder, it would be quite complex since you would have to use a temporary table and migrate the data as well. However, if you use the alter table approach in the scheme file, you can simply run the appropriate alter table commands in each environment to get the schema up to date.

Then it's easy to see when merging PRs what db scripts have to be run. Those can then be copied over to the `database/manual-updates` folder so we have a record of what changes have been applied and when. A lot easier to view them that way than working it out from the repo history.

The schema file is also used in testing to set up the entire database which is a good way to keep the db schema as a whole in check. If we can create the database at any time by running the entire file and it's written in migration style, we know the changes have been applied correctly.

As for the seeds file, that was created so we could easily populate test databases with some basic data for testing the app and setting up local dev environments. It has no other purpose at the moment.